package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Mar 12 15:55:40 UTC 2019 - Dominique Leuenberger <dimstar@opensuse.org>
+
+- Default to wicked on generic desktop: do the same as we did
+  before yast learned to auto-pick NetworkManager if the
+  control.xml happened to request it (bsc#1127228).
```

Comment:
We usually add also the version in the changelog. So I miss the version increase in the .spec file and here. Or isn't it needed?